src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -122,6 +126,12 @@ function MonacoInstance ({logFileState, changeStateCallback, loadingLogs, logDat
      * @param {object} monaco
      */
     const handleEditorDidMount =(editor, monaco) => {
+        // Restore "pageSize" cleared in handleEditorWillMount()
+        //  if no OOM happens
+        setTimeout(()=>{
+            localStorage.setItem("pageSize", logFileState.pageSize);
+        }, 0);
```

Comment:
```suggestion
        // No OOM occurred, so restore the "pageSize" value that was cleared in
        // handleEditorWillMount().
        setTimeout(() => {
            localStorage.setItem("pageSize", logFileState.pageSize);
        }, 0);
```

Reply:
Why wrap this in a setTimeout?

Reply:
My bad - I misunderstood where the crash happens. I was thinking about scheduling the "restore" to be run after `onMount`, but after testing a few times it has become clear that the OOM hapeens when the `editorRef.current.setValue(logData)` statement (see below) is executed.

Reply:
We can add a callback to wrap this single line and invoke the callback as the very last thing in this `onMount` handler.

Reply:
Ah gotcha, sounds good.